package.json

```diff
@@ -20,11 +20,6 @@
     "storybook": "start-storybook -p 6006 -s public",
     "storybook:build": "build-storybook -s public"
   },
-  "husky": {
```

Comment:
This is taking way too long, we can bring it back if we make it lint-staged like watchparty does

Reply:
Sure thing!